vars/patchDockerImage.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+void call(Map args = [:]) {
+    def lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
+    String docker_image = "opensearchproject/${args.product}:${args.tag}"
+    boolean tag_latest = false
+
+    if (args.tag == "2"){
+        tag_latest = true
+    }
```

Comment:
This is so bad can we please dont hardcode 2 as latest?
We should retrieve the `latest` tag and get the major version instead.
Thanks.